.devcontainer/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:20.04
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
This ENV command should be removed and env variable declaration added on line 9 before apt-get per Docker's [official docs](https://docs.docker.com/engine/faq/#why-is-debian_frontendnoninteractive-discouraged-in-dockerfiles)